lib/KSelect/index.vue

```diff
@@ -255,6 +261,9 @@
         type: String,
         default: '',
       },
+      /**
+       * key attribute to identify individual elements within a list rendered
```

Comment:
Maybe this can change to 

`* Used as a key identifier for individual elements in a rendered list.`